en/controllers/components.rst

```diff
@@ -37,39 +37,35 @@ Controller's ``initialize()`` method or via the ``$components`` array::
         public function initialize(): void
         {
             parent::initialize();
-            $this->loadComponent('Auth', [
-                'authorize' => 'Controller',
-                'loginAction' => ['controller' => 'Users', 'action' => 'login']
+            $this->loadComponent('RequestHandler', [
+                'viewClassMap' => ['json' => 'AppJsonView'],
             ]);
             $this->loadComponent('Security', ['blackholeCallback' => 'blackhole']);
         }
 
     }
 
-You can configure components at runtime using the ``config()`` method. Often,
+You can configure components at runtime using the ``setConfig()`` method. Often,
 this is done in your controller's ``beforeFilter()`` method. The above could
 also be expressed as::
 
     public function beforeFilter(EventInterface $event)
     {
-        $this->Auth->config('authorize', ['controller']);
-        $this->Auth->config('loginAction', ['controller' => 'Users', 'action' => 'login']);
-
-        $this->Cookie->config('name', 'CookieMonster');
+        $this->RequestHandler->setConfig('viewClassMap', ['rss' => 'MyRssView']);
     }
 
-Like helpers, components implement a ``config()`` method that is used to get and
+Like helpers, components implement a ``getConfig()`` method that is used to get and
```

Comment:
... components implement ``getConfig()`` and  ``setConfig()`` methods those are used to _get and 
set_ ...

Reply:
Thanks, fixed in a follow up commit to 4.x